llvm/include/llvm/IR/Constants.h

```diff
@@ -289,6 +289,10 @@ class ConstantFP final : public ConstantData {
   /// host double and as the target format.
   static Constant *get(Type *Ty, double V);
 
+#ifdef __FLOAT128__
+  static Constant *get128(Type *Ty, float128 V);
```

Comment:
Just use the APFloat version for float128?

Reply:
Sure, I've removed this function now.